src/ContentHubClient.php

```diff
@@ -620,6 +620,20 @@ public function getWebHook($url)
         return [];
     }
 
+    /**
+     * @param $webhook_uuid
+     *
+     * @return array
+     *
+     * @throws \Exception
+     */
+    public function getInterestsByWebhook($webhook_uuid)
+    {
+        $data = $this->getResponseJson($this->get("/interest/webhook/$webhook_uuid"));
+
+        return $data['data']['interests'];
```

Comment:
What happens if:
1- `$webhook_uuid` is not valid?
2- `$webhook_uuid` does not exist?
3-  $data['data'] has not `interests` set?
4- `$this->get("/interest/webhook/$webhook_uuid")` is not a valid JSON?

Reply:
I've added a try...catch to getResponseJson() and getInterestsByWebhook() will return an empty array now too.

Reply:
Did you check if this was necessary at all by testing the current behavior? Because chances are that these have been already dealt with.

Reply:
I did look at it and it should at minimum cover for any issues with the plexus request... let's say if webhook_uuid is an empty string, etc.... or if something changes unknowingly with the api.